testcases/brasil_4/hydro_dict.jl

```diff
@@ -22,10 +24,12 @@ function hydrogenerator_dict(;
         "initial_volume" => initial_volume,
         "production_factor" => production_factor,
         "spill_cost" => spill_cost,
+        "minimal_outflow_violation_cost" => minimal_outflow_violation_cost,
+        "minimal_volume_violation_cost" => minimal_volume_violation_cost,
         "downstream_turn" => downstream_turn,
         "downstream_spill" => downstream_spill,
     )
-    if name != nothing
+    if name !== nothing

```

Comment:
```suggestion
    if !isnothing(name)
```
Maybe we can already change to `isnothing`.